hello-world/AKS/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ubuntu:22.04
-RUN apt-get -y update && apt-get -y install nginx python3
+RUN apt-get update && apt-get -y upgrade && apt-get -y install nginx python3.10
```

Comment:
If the layer is cached, this upgrade won't have any effect after this update.  Just a heads up in case of aggressive caching during the build process.  https://docs.docker.com/engine/reference/builder/#cache-invalidation-for-run-instructions

Reply:
I built it with the --no-cache flag